.github/workflows/ci-frontend.yml

```
@@ -0,0 +1,58 @@
+            pnpm lint:ci
+          elif [ "${{ matrix.task }}" == "build" ]; then
+            pnpm build
+          fi
+
+      - name: Upload build artifacts
+        if: matrix.task == 'build'
+        uses: actions/upload-artifact@v4
+        with:
+          name: frontend-build
+          path: frontend/dist/
+          retention-days: 7
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)